src/ott/initializers/linear/initializers.py

```diff
@@ -313,3 +338,80 @@ def body_fn(i: int, f: jnp.ndarray) -> jnp.ndarray:
     return f.at[i].set(new_f)
 
   return jax.lax.fori_loop(0, len(f), body_fn, f)
+
+
+@jax.tree_util.register_pytree_node_class
+class SubsampleInitializer(DefaultInitializer):
+  """Subsample initializer :cite:`thornton2022rethinking:22`.
+
+  Subsample each :class:~ott.geometry.point_cloud.PointCloud, then compute closed from
+  Sinkhorn potential between subsampled approximations. Use this potential
+  to initialize Sinkhorn potentials/scalings.
+
+  Args:
+    subsample_n: number of points to subsample from each point cloud.
+  """
+
+  def __init__(
+      self,
+      subsample_n: int,
```

Comment:
Can we make this argument name a bit smaller? Like `n_x` or similar? Not really a huge issue.

Reply:
I quite like `subsample_n`, I think it's explicit for an argument.

Reply:
Ok, but would at least make it more consistent and rename `subsample_n_x`.

Reply:
Please also add docstrings for the 2 missing parameters.

Reply:
Ideally I wanted `subsample_n`  to act as a single argument for `x` and `y`. In most cases it's only worth using if size of point clouds for `x` and `y` are very large. 

But if you prefer subample_n_x then sure.